### DISCUSSION TITLE
J'ai un problème de flash de ma carte ESP32-cam

### DISCUSSION THREAD

**Discussion by GAKFANDANG** [micropython/micropython#14379] [ESP32]
Salut à tout le monde ! Je viens auprès de vous parceque j'ai un problème si quelqu'un peut m'aider ça me sortira du gouffre.
Mon problème est que j'ai acheté une carte ESP32-cam mais pour flasher le micropython sur cette carte, je rencontre de tas de soucis, je suis sur l'IDE thonny de python mais quand je charge le fichier.bin sur ma carte, je constate que dans la console une sorte de défilement sans fin de mots que je ne comprends pas. S'il vous plaît aidez-moi je vous en prie merci !

**Comment by fdufnews** [comment]
If you want more people to answer your questions, you'd better switch to english and give a little more details on what you've done.
What version of Micropython have you flashed on your board?
What the message that is scolling on your terminal?

Si tu veux avoir plus de réponses tu ferrais mieux de te mettre à l'anglais et de donner un peu plus de détails sur ce que tu as fait.
Quelle version de Micropyhon as-tu programmé?
Quel est le message qui défile sur l'ècran?

**Reply by GAKFANDANG** [reply]
OK thanks ! It is with thonny4.1.4 that I want to flash the micropython on my ESP32-CAM card and when I load the file and start flashing, it shows me the following scrolling message constantly:
"load:0x4008400,len:3340
entry 0x40080618
, len:13816"

**Reply by fdufnews** [reply]
Maybe this can be of some interest for you
https://www.instructables.com/How-to-Use-ESP32-CAM-With-MicroPython/
